src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj

```diff
@@ -11,6 +11,14 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net462' ">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="nlog.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="nlog.config">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
I did that from Visual studio. I think that the effect is to "remove it from the default rule, and then add a special case rule".